docs/pack/kubernetes/sync.md

```diff
@@ -218,6 +218,108 @@ slight load on the Kubernetes master.  Note that applying the all the
 resource specs in a sync repository can take some time, so do not set
 the synchronization interval too be too short.
 
+## Fetching resources
+
+If you already have a Kubernetes cluster with many resources in it, it
+can be daunting to get started on your GitOps journey.  The Atomist
+CLI provides the `kube-fetch` command to assist in fetching resources
+from a Kubernetes cluster and writing their resource specs to files.
+
+```
+atomist kube-fetch
+```
+
+The above command will use your currently configured Kubernetes
+credentials to fetch a default set of resources, excluding those
+typically managed by Kubernetes itself, remove common read-only
+properties populated by the Kubernetes system, and write the resulting
+resource specs to YAML files in the current directory.  Once you have
+fetched the resource specs and modified them to suit your needs, you
+can commit them to your sync repository and fully embrace GitOps.
+
+If you prefer the resource spec files are in JSON format, you can use
+the `--output-format` command-line option to select the JSON format.
+
+```
+atomist kube-fetch --output-format=json
+```
+
+If you would like the resource spec files written to a different
+directory, you can use the `--output-dir` command-line option to
+provide a different output directory.
+
+```
+atomist kube-fetch --output-dir=OUTPUT_DIR
+```
+
+If you would like the Kubernetes secret resource data property values
+to be encrypted as described above, you can use the `--secret-key`
+command-line option to provide the same encryption key your SDM will
+use.
+
+```
+atomist kube-fetch --secret-key=SECRET_KEY
+```
+
+If you would like to customize the set of resources fetched from the
+Kubernetes cluster, you can use the `--options-file` command-line
+option to provide the path to a file containing the selection options
+for which resources to fetch.
+
+```
+atomist kube-fetch --options-file=OPTIONS_FILE
+```
+
+The `OPTIONS_FILE` should contain a JSON object with the following
+structure.
```

Comment:
is there an options file they could start from, that we could link?

Reply:
No, but I could include an example. The default options include selectors with the `filter` property, which is a function and therefore cannot be specified in a JSON object read from a file.

Reply:
That reminds me, you can't include regular expressions in a JSON object either.

Reply:
I've added an example and removed the regular expression stuff.